cli/flags.rs

```diff
@@ -510,7 +512,10 @@ To execute a script:
 To evaluate code in the shell:
 
   deno eval \"console.log(30933 + 404)\"
-";
+",
+    crate::version::deno()
```

Comment:
This won't work for canary releases, because `version::deno()` returns semver + 7 characters from git hash. I think we should always split version on `+` sign to get latest version and call it a day

Reply:
Alright, didn't know about this for canary version. Will fix it!

Reply:
Should be fixed now